apolloschurchapp/src/tabs/home/Features/ActionListCardFeature/index.js

```diff
@@ -0,0 +1,56 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+
+import { styled, ActionListCard, H3, H6 } from '@apollosproject/ui-kit';
+
+const Title = styled(
+  ({ theme }) => ({
+    color: theme.colors.text.tertiary,
+  }),
+  'ActionListCardFeature.Title'
+)(H6);
+
+const Subtitle = styled({}, 'VerticalCardListFeature.Subtitle')(H3);
```

Comment:
Should we be referencing the VeritcalCardList or the ActionList here?

Reply:
opps copy and paste error. fixed in fec4ac4